implementations/rust/ockam/ockam_command/src/enroll/auth0.rs

```diff
@@ -73,6 +76,13 @@ async fn enroll(
 
 pub struct Auth0Service;
 
+impl Auth0Service {
+    const DOMAIN: &'static str = "dev-w5hdnpc2.us.auth0.com";
```

Comment:
It's not great to expose our auth0 domain more than we need to. Can we open a ticket to remove this from the source code in the near future and move it to something we pass in via deployment configuration?